packages/web/app/src/pages/target-history.tsx

```diff
@@ -212,7 +212,7 @@ function HistoryPageContent(props: {
       projectId={props.projectId}
       targetId={props.targetId}
       page={Page.History}
-      className="h-full"
+      className="h-[--hive-content-height] overflow-scroll"
```

Comment:
scrolls in only content

https://github.com/kamilkisiela/graphql-hive/assets/7361780/edaab028-dbe2-4c19-a130-15454c77f9b7